Task 2-1/main.cpp

```diff
@@ -0,0 +1,84 @@
+#define _USE_MATH_DEFINES //for C++
+#define  M_PI
+
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+/*
+*\ brief Находит третью сторону треугольника
+*\ param firstSide Длина первой стороны треугольника
+*\ param sekondSide Длина второй стороны треугольника
+*\ param rad Угол между этими сторонами
+*\ return Длина третьей стороны треугольника
+*/
+double getThirdSideTriangle(double firstSide, double sekondSide, double rad);
+
+/*
+* \brief Находит площадь треугольника
+* \param firstSide Длина первой стороны треугольника
+* \param sekondSide Длина второй стороны треугольника
+* \param  rad Угол между этими сторонами
+* \rerurn Площадь треугольника
+*/
+double getAreaTriangle(double firstSide, double sekondSide, double rad);
+
+/*
+* \brief Находит радиус описанной окружности 
+* \param firstSide Длина первой стороны треугольника
+* \param sekondSide Длина второй стороны треугольника
+* \param thirdSide Длина третьей стороны треугольника
+* \param area Площадь треугольника
+* \return Площадь треугольника
+*/
+double getRadiusCircumscribedCircle(double thirdside, double rad);
+
+/*
+* \brief Переводит градусы в радианы
+* \param angle Градусная мера угла
+* \return Угол в радианах
+*/
+double getToRadians(double angle);
+ 
+/*
+* \brief Вход в программу
+* \return в случае успеха возвращает 0
+*/
+
+int main()
+{
+	double firstSide, sekondSide, angle;
+	cout << "Length first side: "; cin >> firstSide;
+	cout << "Length sekond side: "; cin >> sekondSide;
+	cout << "Angle between the sides: "; cin >> angle;
+
+	const double rad = getToRadians(angle);
+	const double thirdSide = getThirdSideTriangle(firstSide, sekondSide, rad);
+	const double area = getAreaTriangle(firstSide, sekondSide, rad);
+	const double radius = getRadiusCircumscribedCircle(thirdSide, rad);
+
+	cout << "Length of the third side: " << thirdSide << ", Area of the triangle: " << area << ", Radius of the circumscribed circle: " << radius;
+
+	return 0;
+}
+
+double getThirdSideTriangle(double firstSide, double sekondSide, double rad)
+{
+	return sqrt(pow(firstSide, 2) + pow(sekondSide, 2) - 2 * firstSide * sekondSide * cos(rad)); //Третью сторону треугольника находим по теореме косинусов
+}
+
+double getAreaTriangle(double firstSide, double sekondSide, double rad)
```

Comment:
Если параметры функции не изменяются, то они явно должны быть обозначены `const`
[By Alex Allain](https://www.cprogramming.com/tutorial/const_correctness.html)
>The const keyword allows you to specify whether or not a variable is modifiable. You can use const to prevent modifications to variables and const pointers and const references prevent changing the data pointed to (or referenced).
>Const дает вам возможность более четко документировать вашу программу и фактически обеспечивать соблюдение этой документации. Ключевое слово const обеспечивает вашим пользователям гарантии, позволяющие оптимизировать производительность без угрозы повреждения их данных, обеспечивая соблюдение вашей документации. Например, константные ссылки позволяют указать, что упомянутые данные не будут изменены; это означает, что вы можете использовать константные ссылки как простой и немедленный способ повышения производительности для любой функции, которая в настоящее время принимает объекты по значению, не беспокоясь о том, что ваша функция может изменить данные. Даже если это произойдет, компилятор предотвратит компиляцию кода и предупредит вас о проблеме. С другой стороны, если бы вы не использовали константные ссылки, у вас не было бы простого способа гарантировать, что ваши данные не были изменены.